content.js

```diff
@@ -230,8 +230,10 @@ function resetValues() {
 }
 
 function videoIDChange(id) {
+  if(id === false) return false;
+  videoID = id; // Global
   //not a url change
-  if (sponsorVideoID == id) return;
+  if (sponsorVideoID == videoID) return;
```

Comment:
`sponsorVideoID` and `videoID`? A little confusing having both.

Reply:
There both called videoID now the id check is no longer using videoID